Common/src/Storage/TaskTableExtensions.cs

```diff
@@ -22,19 +22,22 @@
 using System.Threading.Tasks;
 
 using ArmoniK.Api.gRPC.V1.Submitter;
+using ArmoniK.Core.Common.Exceptions;
+using ArmoniK.Core.Common.gRPC;
 
 using TaskStatus = ArmoniK.Api.gRPC.V1.TaskStatus;
 
 namespace ArmoniK.Core.Common.Storage;
 
 public static class TaskTableExtensions
 {
-  public static Task<int> CancelTasks(this ITaskTable   taskTable,
-                                      TaskFilter        filter,
-                                      CancellationToken cancellationToken = default)
-    => taskTable.UpdateAllTaskStatusAsync(filter,
-                                          TaskStatus.Cancelling,
-                                          cancellationToken);
+  public static async Task<int> CancelTasks(this ITaskTable   taskTable,
+                                            TaskFilter        filter,
+                                            CancellationToken cancellationToken = default)
+    => (int)await taskTable.UpdateAllTaskStatusAsync(filter,
+                                                     TaskStatus.Cancelling,
+                                                     cancellationToken)
+                           .ConfigureAwait(false);
```

Comment:
This looks like a duplicate of CancelTasksAsync

Reply:
It is not the same filtering in input. TaskFilter is a gRPC object we convert to expression